scss/progressCard.scss

```diff
@@ -0,0 +1,49 @@
+.progress-card__container {
+  width: 350px;
```

Comment:
It should be `100%` or auto. Specifying a defined pixel causes the progress-body card to extend pass the column.